src/routes/Post.js

```diff
@@ -12,14 +13,21 @@ export default function Post() {
 
       <ul className="nav nav-tabs mb-3">
         <li className="nav-item">
-          <Link to={`/posts/${post.id}/comments`} className="nav-link">
+          <NavLink
+            to={`/posts/${post.id}/comments`}
+            className={isLinkActive}
+            end
```

Comment:
> If the end prop is used, it will ensure this component isn't matched as "active" when its descendant paths are matched.

https://reactrouter.com/en/main/components/nav-link